webapps/host-manager/css/manager.css

```diff
@@ -73,6 +79,7 @@ td.title {
     font-style    : italic;
     font-weight   : bold;
     background    : #D2A41C;
+	border-radius : 10px 10px 0 0;
```

Comment:
Use 4 spaces rather than 1 tab for indents. This comment applies globally, not just to this line.